src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -97,51 +101,114 @@ class AmendmentTableImpl : public AmendmentTable
     amendmentList_t getDesired();    // amendments we support, do not veto, are not enabled
 };
 
-void
-AmendmentTableImpl::addInitial ()
+namespace detail
```

Comment:
@vinniefalco I disagree with your statement that detail namespaces are for header files only.  In a previous life I made extensive use of anonymous namespaces in cpp files.  Our unity builds make anonymous namespaces ineffective.  In our code base I have replaced using anonymous namespaces with `detail` namespaces in cpp files.  In effect, I'm hand-building a pseudo-anonymous namespace.  Can you suggest an alternative?